src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -70,6 +70,8 @@
  * @checkstyle VisibilityModifierCheck (500 lines)
  * @checkstyle JavadocVariableCheck (500 lines)
  * @checkstyle MagicNumberCheck (500 lines)
+ * @todo #90:30min OCC metric: need to implement the rest of the test cases.
```

Comment:
@llorllale same question, what does "no time" mean? Had trouble with the other test cases, did they fail?